community/import-tool/src/main/java/org/neo4j/tooling/ImportTool.java

```diff
@@ -645,4 +655,35 @@ static void warn( String warning )
     {
         System.err.println( warning );
     }
+
+    private enum ManualPage
+    {
+        IMPORT_TOOL_FORMAT("import-tool-header-format.html");
+
+        private String page;
+
+        ManualPage( String page )
+        {
+            this.page = page;
+        }
+
+        public String getReference(Anchor anchor)
+        {
+            return page + "#" + anchor.anchor;
+        }
+    }
+
+    private enum Anchor
+    {
+        ID_SPACES("_id_spaces"),
```

Comment:
The leading underscore means the anchor id was generated from the heading, which might get changed.
It's better to set an id on the target like so: https://github.com/nawroth/neo4j/commit/c1a85236c8e0dfb21d956f175d1934eb78c2a89a
and use that instead.